Resources/Prototypes/Maps/cluster.yml

```diff
@@ -2,7 +2,7 @@
   id: Cluster
   mapName: 'Cluster'
   mapPath: /Maps/cluster.yml
-  minPlayers: 10
+  minPlayers: 0
```

Comment:
clusters lowpop job requirements are even worse: atlas only requires an engineer to refill plasma tanks, cluster requires an engineer to refuel the ame and a cargo tech to make money for more fuel